src/components/Switch/Switch.module.scss

```diff
@@ -0,0 +1,75 @@
+﻿@import "../../styles/mixins";
+
+
+.switch-input {
+  display: inline-block;
```

Comment:
The styling here doesn't match the styling of the Admin UI. Take a look at our Pattern Library and copy these styles. https://stage.makaira.io/pali/?path=/story/atoms-form-elements-switch--with-title